qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -1453,6 +1467,11 @@ def save(
 
         os.chdir(cwd)
 
+    @deprecate_func(
+        since="0.6",
+        additional_msg="Schedule serialization is natively performed in QPY format.",
```

Comment:
```suggestion
        additional_msg="Saving calibraiton in csv format is deprecate as well as functions that support this functionality.",
```
Let's just outright say we deprecate CSV.